<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Hemera::Internal::HemeraRunConfigurationWidget</class>
 <widget class="QWidget" name="Hemera::Internal::HemeraRunConfigurationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QRadioButton" name="simpleRadio">
     <property name="text">
      <string>Run in simple mode</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Application ID:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="applicationIdEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Orbit Handler:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="starCombo"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QRadioButton" name="advancedRadio">
     <property name="text">
      <string>Run in advanced mode</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Applications:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="applicationsLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Features:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QListView" name="featuresView"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>simpleRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>applicationIdEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>simpleRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>starCombo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>simpleRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>applicationsLineEdit</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>simpleRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>featuresView</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
